src/main/java/fr/eirb/lemondedenemo/periscope/display/FishTankDisplay.java

```diff
@@ -0,0 +1,145 @@
+package fr.eirb.lemondedenemo.periscope.display;
+
+import fr.eirb.lemondedenemo.periscope.utils.Coords;
+import fr.eirb.lemondedenemo.periscope.utils.Fish;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.List;
+import javafx.animation.TranslateTransition;
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.Pane;
+import javafx.scene.paint.Color;
+import javafx.stage.Stage;
+import javafx.util.Duration;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class FishTankDisplay extends Application implements TankDisplay {
+
+  private static FishTankDisplay instance;
+  private static Logger logger;
+  private final HashMap<String, FishItem> fishes = new HashMap<>();
```

Comment:
Use Map as apparent type for fishes.
```suggestion
import java.util.Map;

  private final Map<String, FishItem> fishes = new HashMap<>();
```